### DISCUSSION TITLE
@types/source-map-support missing 'state' parameter on 'wrapCallSite'

### DISCUSSION THREAD

**Discussion by watsonsong** [DefinitelyTyped/DefinitelyTyped#60483] [Issues with a @types package]
From the [index.d.ts](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/source-map-support/index.d.ts#L42), the wrapCallSite is:
`export function wrapCallSite(frame: any /* StackFrame */): any /* StackFrame */;`

But the [source-map-support.js](https://github.com/evanw/node-source-map-support/blob/master/source-map-support.js#L368) has a state parameter, which type is an object combine 'nextPosition' and 'curPosition'.

[a comment by an automated account is not shown]

**Comment by BendingBender** [comment]
See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61841
